schemas/stsci.edu/asdf/0.1.0/asdf-schema.yaml

```diff
@@ -0,0 +1,46 @@
+%YAML 1.1
+---
+$schema: "http://json-schema.org/draft-04/schema"
+id: "http://stsci.edu/schemas/asdf/0.1.0/asdf-schema"
+title:
+  ASDF schema
+description: |
+  A metaschema extending YAML Schema and JSON Schema to add support
+  for some ASDF-specific checks, related to nd-arrays.
```

Comment:
Doh.  Not sure why I didn't see that with `ndim` and `max_ndim`.

`datatype` requires more complex logic though -- since (at least for the JWST pipeline use case), we want to be able to say things like "at least 32 bit integers" rather than an exact match.  If we expressed datatypes numerically (not as strings) then that might be possible.  But there's still the corner case of float-to-int casting where it's a little more complicated than just the number of bits.